test/animations.jl

```diff
@@ -715,3 +715,52 @@ end
     Object(1:10, (args...) -> circle(O, 50, :fill))
     @test_logs (:error,) render(video; pathname = "test.mp3")
 end
+
+@testset "test layer vs nonlayer actions" begin
+    function act_all!(a)
+        act!(a, Action(1:70, anim_translate(Point(50, 50))))
+        act!(a, Action(71:140, anim_translate(Point(50, 50), Point(-50, -50))))
+    end
+
+    n_frames = 257
+    nolayer_video = Video(500, 500)
+    Background(1:n_frames, ground)
+    circ = Object(JCircle(O, 20, action = :fill, color = "white"))
+    act_all!(circ)
+    render(nolayer_video, tempdirectory = "images/without_layer", pathname = "")
+
+    layer_video = Video(500, 500)
+    Background(1:n_frames, ground)
+    l1 = @JLayer 1:n_frames begin
+        Object(JCircle(O, 20, action = :fill, color = "white"))
+    end
+    act_all!(l1)
+    render(layer_video, tempdirectory = "images/with_layer", pathname = "")
+
+    @test_reference "images/with_layer/0000000008.png" load(
```

Comment:
do you mind looping over them instead? :wink:

Reply:
sorry I don't understand what I should loop the tests? I add now the actions at the same time though it look like if I try that with anim_rotate_around it doesn't work, with anim_rotate it does

Reply:
Just 
```julia
for frame in [8, 16, 32, 64, 128, 256]
    png_name = lpad(string(frame), 10, "0")
    @test_reference "images/with_layer/$png_name.png" load(
        "images/without_layer/$png_name.png",
    )
end
```
That should work I hope

Reply:
sure will do didn't understand